android/app/src/main/AndroidManifest.xml

```diff
@@ -1,11 +1,15 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.zooniversemobile"
     android:versionCode="1"
-    android:versionName="1.0">
+    android:versionName="1.0"
+    xmlns:tools="http://schemas.android.com/tools">
 
     <uses-permission android:name="android.permission.INTERNET" />
-    <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW"/>
+    <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW" tools:remove="${excludeSystemAlertWindowPermission}"/>
```

Comment:
Yes, the "remove" part will only happen in release - so that the SYSTEM_ALERT_WINDOW is only for debug.  I can rewrite this to make more sense, since it's a double-negative now?